solutions/advanced/ac-GameOnTree.mdx

```diff
@@ -71,4 +71,38 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
```

Comment:
add a warning saying you have to sumbit with pypy

Reply:
added